src/app/wallets/intraledger-send-payment.ts

```diff
@@ -157,15 +157,19 @@ const executePaymentViaIntraledger = async ({
     amount,
     walletId,
   })
+
   if (intraledgerLimitCheck instanceof Error) return intraledgerLimitCheck
 
   const recipientUser = await UsersRepository().findByUsername(recipientUsername)
   if (recipientUser instanceof Error) return recipientUser
   if (recipientUser.id === userId) return new SelfPaymentError()
+  // FIXME: selfPayment should be at the walletId level, no longer at the UserId
+  // but this is not a bloquer until we have multiple wallets per account
```

Comment:
Typo in `blocker` and I prefer to have comments above the line that they refer to....

no big deal.

Reply:
oh yeah I had used the french version lol